src/workloads/networking/TransportLayerConnectTiming.yml

```diff
@@ -0,0 +1,31 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/security"
+
```

Comment:
Consider adding `Description: this workload does....` for future maintainers / build-barons.

Reply:
Done